src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -591,8 +591,8 @@ public override void ExecuteCmdlet()
                     }
                 });
 
-                WriteInformation($"{Resources.AnnouncementsHeader}{System.Environment.NewLine}{Resources.AnnouncementsMessage}{System.Environment.NewLine}");
-                WriteInformation($"{Resources.ReportIssue}{System.Environment.NewLine}");
+                WriteInteractiveInformation($"{Resources.AnnouncementsHeader}{System.Environment.NewLine}{Resources.AnnouncementsMessage}{System.Environment.NewLine}");
```

Comment:
The original method called does not pass Newline as False to the `WriteInformation` method. But after refactor it starts to pass `false`. Will this cause the `{System.Environment.NewLine}` not work?